x/qgb/orchestrator/relayer.go

```diff
@@ -147,6 +147,22 @@ func (r *Relayer) submitDataRootTupleRoot(
 	commitment string,
 	confirms []types.MsgDataCommitmentConfirm,
 ) error {
+	// sanitizing the confirms commitments
+	for _, conf := range confirms {
+		if conf.Commitment != commitment {
+			r.logger.Error("confirms don't have the same commitment",
+				"expected_commitment",
+				commitment,
+				"confirm_commitment",
+				conf.Commitment,
+				"nonce",
+				conf.Nonce,
+				"signer_eth_address",
+				conf.EthAddress)
+			return ErrInvalidCommitmentInConfirm
```

Comment:
@evan-forbes What do you think makes more sense:
- not relay attestations where one or more data commitments have an invalid commitment?
- relay if we have enough data commitments that are good (without considering the faulty ones).

I guess we can go for second option for now. 
What do you think?

Reply:
Agree that option 2 is probably best. We want to ignore potentially malicious validators. In the future, we will slash for this, as they sign an incorrect data commitment.

Reply:
Awesome. Thanks a lot.